defaults/initrd.scripts

```diff
@@ -2054,8 +2130,18 @@ openLUKS() {
 					else
 						[ -e /dev/tty ] && run mv /dev/tty /dev/tty.org
 						run mknod /dev/tty c 5 1
+						ASK_PASS=1
```

Comment:
Why here? '1' means don't ask pwd.

Reply:
I had assumed the default was to NOT ask, meaning 1 meant TO ask.

Reply:
By default is to ask.

Reply:
Ah, ok. My mistake. It's cannot be interprete as return code. Then please add also addidtional ASK_PASS=1
```
if [ -n "${LUKS_KEY}" ]
then
...
else
ASK_PASS=1
```